features/cancel-meeting.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Cancel meeting
+  As the host of a meeting
+  I want to be able to cancel a meeting
+  So that I can let the other participants know the meeting can't go through
+
+  Scenario: Host cancels prospective meeting
```

Comment:
I'll split this up and add this scenario in a separate pull request, so we can wait with merging that until a future version.